gravitee-apim-e2e/api-test/src/apis/settings/mapi-v1/notification-setting.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { API_USER, forManagementAsAdminUser } from '@gravitee/utils/configuration';
+import { RolesApi } from '../../../../../lib/management-webclient-sdk/src/lib/apis/RolesApi';
+import { UsersApi } from '../../../../../lib/management-webclient-sdk/src/lib/apis/UsersApi';
+import { RoleScope, UserEntity } from '../../../../../lib/management-webclient-sdk/src/lib/models';
+import { describe, expect } from '@jest/globals';
+import { noContent, succeed } from '@lib/jest-utils';
+import { ConfigurationApi } from '../../../../../lib/management-webclient-sdk/src/lib/apis/ConfigurationApi';
+
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+
+const managementRolesResourceAsAdminUser = new RolesApi(forManagementAsAdminUser());
+const managementUserResourceAsAdminUser = new UsersApi(forManagementAsAdminUser());
+const configurationResourceAsAdmin = new ConfigurationApi(forManagementAsAdminUser());
+
+describe('Notification settings tests', () => {
```

Comment:
I just added the run e2e label on the PR to check that everything is ok before merging

Reply:
It was missing the license but now it's working :)